curriculum/challenges/espanol/11-machine-learning-with-python/tensorflow/neural-networks-activation-functions.md

```diff
@@ -10,7 +10,7 @@ dashedName: neural-networks-activation-functions
 
 ## --text--
 
-Which activation function switches values between -1 and 1?
+Which activation function squishes values between -1 and 1?
```

Comment:
there is no need to change spanish translation, crowdin will take care of this

Reply:
Should I remove the commit which changes spanish translation?

Reply:
Yes, reverting the changes in Spanish translation, should fix https://github.com/freeCodeCamp/freeCodeCamp/pull/46913#issuecomment-1185782646